script/build-task.js

```diff
@@ -70,6 +70,11 @@ export const buildTask = async (
 				},
 				*/
 				// endregion
+				{
+					id: "libWrapper",
```

Comment:
this is already a Plutonium dep (albeit a """soft""" one), so no real change here. Declaring transitive dependencies is Good:tm:

Reply:
Isn't plutonium's libwrapper dependency actually enforced now? Is Plut still using the shim or is is relying upon libwrapper being present?

Reply:
> plutonium's libwrapper dependency actually enforced now
not as part of the manifest, no, but it does crash on load and tells you that you need to install libWrapper

this PR makes libWrapper a dependency of PAA, therefore we should declare it as a dependency of PAA. The "no real change" bit is just me saying that this doesn't have any practical effects, i.e. we don't need to worry about bricking someone's PAA install, as they're guaranteed to already be running libWrapper